webapi/src/main/scala/org/knora/webapi/instrumentation/metrics/PrometheusServer.scala

```diff
@@ -0,0 +1,70 @@
+package org.knora.webapi.instrumentation.metrics
+
+import zhttp.html._
+import zhttp.http._
+import zhttp.service.EventLoopGroup
+import zhttp.service.Server
+import zhttp.service.server.ServerChannelFactory
+import zio.Runtime
+import zio.ZIO
+import zio.ZLayer
+import zio.metrics.connectors.MetricsConfig
+import zio.metrics.connectors.prometheus
+import zio.metrics.connectors.prometheus.PrometheusPublisher
+import zio.metrics.jvm.DefaultJvmMetrics
+
+import org.knora.webapi.config.AppConfig
+
+object PrometheusServer {
+
+  private val nThreads = 5
```

Comment:
why is this not coming from the config?

Reply:
I'm not sure what it does exactly. I still need to figure that out first. Maybe we don't need it.